starter-code/scripts/article.js

```diff
@@ -34,6 +34,11 @@ Article.prototype.toHtml = function() {
     3. article title,
     4. article body, and
     5. publication date. */
+  $newArticle.find('.byline a').text(this.author);
+  $newArticle.find('.byline a').attr('href', this.authorUrl);
+  $newArticle.find('h1').text(this.title);
+  $newArticle.find('.article-body').html(this.body);
+  $newArticle.find('.datetime').text(this.publishedOn);
```

Comment:
This won't actually find an element to update. Take a closer look at the published part of the template.